ui/pages/smart-transactions/smart-transaction-status-page/index.scss

```diff
@@ -31,4 +37,27 @@
     width: 100%;
     text-align: left;
   }
+
+  &__background-animation {
+    position: relative;
+    left: -88px;
+    background-repeat: repeat;
+    background-position: 0 0;
+
+    &--top {
+      width: 1634px;
```

Comment:
For a future PR:
Where do these constants come from? Should we use computed values or constants?
cc @georgewrmarshall from DS team.

Reply:
Thanks for the tag @dbrans! I tend to think arbitrary values in styles are code smells, but sometimes they are necessary. It would be great to see a screencast of these changes in the PR description so I can review without having to checkout the branch and locate where these changes are implemented @dan437  🙏

Reply:
This width is twice the size of the transaction-background-top.svg image (817px) and I believe it was used in the past to make the animation work properly. We just reused the same code for animation from Smart Swaps as a short term solution. We will be replacing it with a long term solution once it's designed.